### ISSUE TITLE
Allow specifying user email

### ISSUE THREAD

**Issue by sugdyzhekov** [BurdaForward/terraform-provider-aws-sso-scim#29]
### Terraform Version
```
Terraform v1.1.4
on darwin_amd64
+ provider registry.terraform.io/burdaforward/aws-sso-scim v0.5.0
```

### Affected Resource(s)
Please list the resources as a list, for example:
- aws-sso-scim_user

### Expected Behavior
The provider supports the Email field for the user's profile.

### Actual Behavior
It's not possible to specify user's email. Please see the attached screenshot for details.
<img width="1076" alt="Screenshot 2022-02-21 at 19 45 14" src="https://user-images.githubusercontent.com/8955225/155012042-495d755b-92c3-4fba-b8e1-2497b9e87dc1.png">

**Comment by JanKoppe** [comment]
https://github.com/BurdaForward/terraform-provider-aws-sso-scim/commit/4f24d90e6e5c7042b9b05055ff20cdf21a5d2eef
https://github.com/BurdaForward/terraform-provider-aws-sso-scim/commit/835c82c586c39b225fd09b4b878152ea5f6120f1

does that work for you?

**Comment by Volatus** [comment]
> Hi @Volatus,
> 
> 
> 
> https://github.com/BurdaForward/terraform-provider-aws-sso-scim/commit/4f24d90e6e5c7042b9b05055ff20cdf21a5d2eef - user datasource: expose more attributes 
> 
> https://github.com/BurdaForward/terraform-provider-aws-sso-scim/commit/835c82c586c39b225fd09b4b878152ea5f6120f1 - user resource: support setting email attribute
> 
> 
> 
> does that work for you?

Yes, i think so. Thanks

**Comment by JanKoppe** [comment]
Released with v0.7.0
